src/Native/Unix/System.Native/pal_uid.c

```diff
@@ -19,14 +19,14 @@ static int32_t ConvertNativePasswdToPalPasswd(int error, struct passwd* nativePw
     if (error != 0)
     {
         assert(error > 0);
-        *pwd = {}; // managed out param must be initialized
+        memset(&(*pwd), 0, sizeof(Passwd)); // managed out param must be initialized
         return error;
     }
 
     // 0 returned with null result -> entry-not-found
-    if (result == nullptr)
+    if (result == NULL)
     {
-        *pwd = {}; // managed out param must be initialized
+        memset(&(*pwd), 0, sizeof(Passwd)); // managed out param must be initialized
```

Comment:
Ditto

Reply:
Fixed up and rebased over master. Thanks!